public/images/default_preview.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="640" height="640" viewBox="0 0 640 640" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
has this been optimized?

looks pretty janky for a SVG 😄

Reply:
I actually don't know how to, I just exported from Figma that @sarakaandorp provided. If you remember we were in a call doing this 😄.